portable-screencap

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
-set -e
-set -o pipefail
```

Comment:
i recall needing `set -o pipefail` for something. may have been in the `ipfs-screencap` script.